pkg/ssh/ssh.go

```diff
@@ -50,22 +51,30 @@ func SetAuthorizedKeys(cfg *config.CloudConfig, withNet bool) error {
 }
 
 func getKey(key string, withNet bool) (string, error) {
-	if !strings.HasPrefix(key, "github:") {
+	providers := map[string]string{
+		"github": "https://github.com/%s.keys",
+		"gitlab": "https://gitlab.com/%s.keys",
+	}
+
+	url, err := url.Parse(key)
+	if err != nil {
+		return "", err
+	} else if url.Scheme == "" {
 		return key, nil
 	}
 
 	if !withNet {
 		return "", nil
```

Comment:
```suggestion
	if !withNet {
		return "", nil
	}
```
missing bracket is breaking the build

Reply:
🤦

Reply:
🤦